codebuild/e2e_buildspec.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 12
```

Comment:
Can we use a specific version of node here - e.g. `12.15.0`? It'd be good to be explicit so we know exactly whether we need to upgrade due to a security patch. 
All examples in the docs just seem to use a whole nodejs version. https://docs.aws.amazon.com/codebuild/latest/userguide/sample-runtime-versions.html

Reply:
Im not sure that we can, To do that we'd need to run it in our own docker container instead.

Reply:
which might be a good future improvement to this setup

Reply:
Seems like from the docs that currently the runtime version used for nodejs is 12.14, which can pull in a different npm version to the 12.15.0 we use in our application just now. https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html
I think this should be okay for now since I don't imagine there's anything significantly different in these particular npm versions that'd cause our build to be different - however there may later be a mismatch in the package-lock.json on `latest` simorgh and when running `npm ci` on codebuild with a different npm version it may error and fail due to a mismatch. We should open up a ticket to dockerise this sooner than later.